qiling/os/macos/syscall.py

```diff
@@ -338,28 +337,28 @@ def ql_syscall_mmap2_macos(ql, mmap2_addr, mmap2_length, mmap2_prot, mmap2_flags
 
 # 0xca
 def ql_syscall_sysctl(ql, name, namelen, old, oldlenp, new_arg, newlen):
-    ql.nprint("sysctl(name:0x{:X}, namelen:0x{:X}, old:0x{:X}, oldlenp:0x{:X}, new:0x{:X}, newlen:0x{:X})".format(
+    ql.nprint("sysctl(name: 0x%X, namelen: 0x%X, old: 0x%X, oldlenp: 0x%X, new: 0x%X, newlen: 0x%X)" % (
         name, namelen, old, oldlenp, new_arg, newlen
     ))
     ql_definesyscall_return(ql, KERN_SUCCESS)
 
 # 0x112
 def ql_syscall_sysctlbyname(ql, name, namelen, old, oldlenp, new_arg, newlen):
-    ql.nprint("sysctlbyname(name:0x{:X}, namelen:0x{:X}, old:0x{:X}, oldlenp:0x{:X}, new:0x{:X}, newlen:0x{:X})".format(
+    ql.nprint("sysctlbyname(name: 0x%X, namelen: 0x%X, old: 0x%X, oldlenp: 0x%X, new: 0x%X, newlen: 0x%X)" % (
         name, namelen, old, oldlenp, new_arg, newlen
     ))
     ql_definesyscall_return(ql, KERN_SUCCESS)
 
 # 0x126
 # check shared region if avalible , return not ready every time
 def ql_syscall_shared_region_check_np(ql, p, uap, retvalp, *args, **kw):
-    ql.nprint("shared_region_check_np(p: {}, uap: {}, retvalp :{}) : ret:{}".format(p, uap, retvalp, EINVAL))
+    ql.nprint("shared_region_check_np(p: 0x%X, uap: 0x%X, retvalp: 0x%X) : ret: 0x%X" % (p, uap, retvalp, EINVAL))
```

Comment:
Question: should it be 0x%X or 0x%X ?

Reply:
0x%X for uppercase hex and 0x%x for lowercase hex, depends which you prefer

Reply:
since we are all using lower case. make it a standard.

Reply:
how about the parameter names ?
should we print it out or just plain value like shared_region_check_np(X , X , X)

Reply:
i think take out the param too, make it standaized. the param can leave it dprint, just like how u did it previously for those file open